snuba/datasets/storages/outcomes.py

```diff
@@ -62,6 +66,8 @@
         ("outcome", UInt(8)),
         ("reason", String()),
         ("times_seen", UInt(64)),
+        ("quantity", UInt(64, Modifiers(nullable=True))),
```

Comment:
Want to call out here that we decided to make quantity a nullable value and leave it as null if the outcome category is not SESSION or ATTACHMENT. There was also the idea of writing a 1 or a 0, but I didn't see a use case where this quantity value meant anything or would be used for those event types. Let me know if that makes sense.